### DISCUSSION TITLE
RSS feed for Issues

### DISCUSSION THREAD

**Discussion by markcellus** [community/community#7871] [Projects and Issues]
Is there any chance we can get RSS feeds for Issues? 

I know we can get email/web notifications on a repository's issues by clicking Watch button -> Custom -> Issues. However, this auto-subscribes me to all issues and I get notifications for every comment posted to every issue and pollutes my email inbox, which is not my intention. 

I'd like to just see new issues on an RSS feed and manually subscribe to only the issue I would like to get notifications for. In other words, I don't want email notifications about comments on _every_ issue on a repo-only the ones I manually subscribe to from links in the RSS feed. 

Hope that makes sense.

Thanks for any information!

**Comment by kevincox** [comment]
Ideally we could get a feed for any issue search so that we can follow specific labels, words, authors...

**Reply by ghost** [reply]
> Ideally we could get a feed for any issue search so that we can follow specific labels, words, authors...

Agreed.

**Comment by oliroe** [comment]
Agreed this should be offered by GitHub. RSS-Bridge has this functionality in the interim

**Comment by xan2622** [comment]
I second this suggestion.

**Comment by ghost** [comment]
would like an rss feed: pull request/issues/discussions

**Comment by markcellus** [comment]
Hey there! I've made RSS feeds for Github Issues available on [openrss.org](https://openrss.org), a non-profit project I developed to provide RSS feeds for websites that don't have them. 

To get an Issues RSS feed, just navigate to any Issues page and type in `openrss.org/` right before the `github.com` part in the browser URL bar. For example, the RSS feed for the issues at [https://github.com/octokit/octokit.rb/issues](https://github.com/octokit/octokit.rb/issues) is located at [https://openrss.org/github.com/octokit/octokit.rb/issues](https://openrss.org/github.com/octokit/octokit.rb/issues). It may take a little while if it's an issues page that hasn't been cached in the database yet, but should show up after a few seconds. Hope this helps!

**Reply by Merlin04** [reply]
This looks really cool, is there a way to also subscribe to comments on issues?

**Reply by tabacitu** [reply]
OMFG this is excellent. Thank you @markcellus for this, you've saved us!

**Reply by eldy** [reply]
Solution seems to work for full list of issues. It seems not possible to filter however on a tag to get a RSS for only some filtered issues because openrss.org does not support parameter on the url to track.
Example, this is the github issue list I need RSS for:
https://github.com/Dolibarr/dolibarr/issues?q=is%3Aissue+is%3Aclosed+label%3A%22Bug+Security+%28CVE%29%22
There is some parameters.
But openrss.org accept only the url https://github.com/Dolibarr/dolibarr/issues
@markcellus is it possible to make a change in your project so parameters are retreived to forge the url to process ?

**Reply by ghost** [reply]
@markcellus Do you have plans for releasing the source code of openrss.org?

**Reply by ross-spencer** [reply]
+1 it would be great to see more detail on openrss.org.

**Comment by ross-spencer** [comment]
+1 this suggestion. Didn't an RSS feed previously exist for issues?

**Comment by rwp0** [comment]
And for [Pull Requests](https://github.com/orgs/community/discussions/106550) too, please.

**Comment by james-tindal** [comment]
RSSHub. Allows subscribing to all aspects of Github
[https://docs.rsshub.app/routes/programming#repo-issues](https://docs.rsshub.app/routes/programming#repo-issues)

[Blogtrottr](https://blogtrottr.com)
Send new RSS items to an email address.

**Comment by essen** [comment]
Yes please, both issues and pull requests.

**Comment by aldipower** [comment]
Ass openrss.org and rssHub are great, they do not work for private repositories.
So yes, please, GitHub, this functionality would be great!

**Reply by I-I-IT** [reply]
Any RSS feed for private repos would basically make them public so not a good idea. But for public repos, this is really needed

**Reply by kevincox** [reply]
This isn't true. The feeds would have a authentication token in the URL or as a basic auth parameter. See how GitLab does it.

**Comment by hMihaiDavid** [comment]
yes please.

**Comment by rmartine-ias** [comment]
For public repos, https://rss-bridge.org/bridge01/ also works.

**Comment by melygosh** [comment]
+1

**Comment by TalKachman** [comment]
+1
